src/handler.ts

```diff
@@ -19,16 +19,16 @@ interface LineDecorationText {
  * @returns repo and file part of URI.
  */
 export function getParamsFromUriPath(textDocumentURI: string): Params {
-    // TODO: Support more than just GitHub.
+    // TODO: Support more than just GitHub & Gitlab.
     // TODO: Safeguard for cases where repo/fileMatch are null.
     const repoPattern = /(github\.com|gitlab\.com)\/([^\?\#\/]+\/[^\?\#\/]*)/gi
-    const filePattern = /#.*\.(.*)$/gi
+    const filePattern = /#(:?.*\.(.*))$/gi
```

Comment:
Did you mean a non-capture group?
```suggestion
    const filePattern = /#(?:.*\.(.*))$/gi
```
Looks like you are only taking the extension, shouldn't you look at the file path and name too?

Reply:
Ah yes fixed that in my last push. I decided to capture after all, as I want to capture without the `#`.